ibis/impala/kudu_support.py

```diff
@@ -55,7 +55,8 @@ def list_tables(self, filter=''):
     def table_exists(self, name):
         return self.client.table_exists(name)
 
-    def connect(self, host_or_hosts, port_or_ports=7051, rpc_timeout=None):
+    def connect(self, host_or_hosts, port_or_ports=7051,
+                rpc_timeout_ms=None, admin_timeout_ms=None):
```

Comment:
What do you think about leaving the same name for `rpc_timeout` and adding the `admin_timeout` keyword argument? That way we don't break backward compatibility.